### PR TITLE
updated callbacks with params

### PR DESCRIPTION
**Pull request by naresh-n**
Added callback parameters for drag events. Which are useful to determine affected split sections 
in-order to re-render components like charts , graphs and etc.

### CODE REVIEW

File: split.js [hunk 1]
```diff
@@ -211,6 +211,9 @@ var Split = function (ids, options) {
     // | <- this.start                                        this.size -> |
     function drag (e) {
         var offset;
+        var a = elements[this.a].element;
+        var b = elements[this.b].element;
+        var isVertical = e.target.classList.contains("gutter-vertical");
```
Comment (icholy) [inline split.js]: I think you can just pass `direction` instead of `isVertical`. It would be more consistent with the other callback signatures.

**Comment by nathancahill** [comment]
I like the idea, can you separate the maximized styles code in to a different PR?

File: split.js [hunk 2]
```diff
@@ -241,7 +240,7 @@ var Split = function (ids, options) {
         // Call the drag callback continously. Don't do anything too intensive
         // in this callback.
         if (options.onDrag) {
-            options.onDrag(isVertical, a, b);
+            options.onDrag(direction, a, b);
```
Comment (frumbert) [inline split.js]: You could  do `options.onDrag(direction, elements[this.a].element, elements[this.b].element);` and not define the variables unless they were needed, and save a couple of bytes when minified (if you were that pedantic - I guess I must be!)

**Comment by nathancahill** [comment]
`onDragStart` and `onDragEnd` callbacks now have the sizes passed to them.

**Pull request closed without merging**